src/databricks/labs/remorph/reconcile/schema_compare.py

```diff
@@ -107,7 +106,7 @@ def compare(
         self,
         source_schema: list[Schema],
         databricks_schema: list[Schema],
-        source: str,
+        source: Dialects,
```

Comment:
change it to DialectType: `from sqlglot.dialects.dialect import DialectType`

Reply:
done